src/isIsogram.test.js

```diff
@@ -8,12 +8,22 @@ describe(`Function 'isIsogram':`, () => {
   });
 
   it(`should return a boolean`, () => {
+    expect(typeof isIsogram('word')).toBe('boolean');
+  });
 
+  it(`should not be case sensitive`, () => {
+    expect(isIsogram('WORD')).toBe(true);
   });
```

Comment:
You don't check case sensitivity here. To check it, you should have the same letter in uppercase and in lowercase (like `'Aa'`)